### DISCUSSION TITLE
Account blocked

### DISCUSSION THREAD

**Discussion by Iyiola2017** [vercel/vercel#10562] [General]
### Description

Hi everyone, I tried to connect vercel to my github, I couldn't because it was showing me account blocked. Has anyone experienced  such before? and how did you get pass it?

**Comment by jtvberg** [comment]
Same, opened a ticket and I am on day 5 with no answer.

**Reply by Chideraben** [reply]
Am having the same issue as well

**Comment by KILLERLUCIEN** [comment]
same here me to now

**Comment by Chideraben** [comment]
Did you know anyone or how to go about unblocking it?

**Comment by KILLERLUCIEN** [comment]
i am contacting customer care

**Comment by fahim-ahmed05** [comment]
same happened to me 1 day ago. contacted customer care, haven't got any helpful reply
tried signup with mail and github account, they blocked both 🐸

**Reply by fahim-ahmed05** [reply]
they haven't reached out to me or unblocked my account. still waiting!

**Comment by OmotayoGabe** [comment]
Why is my vercel showing blocked i cant login with my github please help

**Comment by MeowFET** [comment]
Same here, 10 days since I received a "Case Opened" email and nothing happened

**Comment by OmotayoGabe** [comment]
please can you help me?

**Comment by Wizzel1** [comment]
Same here, 11 days since I contacted the support without any response
